zuul.d/jobs.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 - job:
```

Comment:
okay, we should now add the new add-build-sshkey vars here, to generate ecdsa keys, and see what breaks.